x-pack/plugins/transform/public/app/sections/create_transform/components/step_define/common/percentiles_agg/config.ts

```diff
@@ -75,9 +103,10 @@ export function getPercentilesAggConfig(
       };
     },
     isValid() {
-      return (
-        typeof this.aggConfig.percents === 'string' && isValidPercentsInput(this.aggConfig.percents)
-      );
+      return validatePercentsInput(this.aggConfig.percents).isValid;
+    },
+    getErrorMessageType() {
```

Comment:
There are two `validatePercentsInput` calls, but I implemented it this way to avoid refactoring all the components to handle the different format of the validation result (e.g., `{isValid, errorMessage}`)